docs/user_guide.md

```diff
@@ -556,76 +531,184 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
 When an image is signed, it has a tick shown in UI; otherwise, a cross sign(X) is displayed instead.  
 ![browse project](img/content_trust.png)
 
-### Vulnerability scanning via Clair 
-**CAUTION: Clair is an optional component, please make sure you have already installed it in your Harbor instance before you go through this section.**
+## Vulnerability Scanning
+
+Harbor provides static analysis of vulnerabilities in images through the open source [Clair](https://github.com/coreos/clair) project. You can also connect Harbor to additional vulnerability scanners by using an interrogation service. The additional scanners that are currently supported are [Anchore](https://anchore.com/harbor/) and [Aqua](https://github.com/aquasecurity/harbor-scanner-aqua).
 
-Static analysis of vulnerabilities is provided through open source project [Clair](https://github.com/coreos/clair). You can initiate scanning on a particular image, or on all images in Harbor. Additionally, you can also set a policy to scan all the images at a specified time everyday.
+It might be necessary to connect Harbor to other scanners for corporate compliance reasons, or because your organization already uses a particular scanner. Different scanners also use different vulnerability databases, capture different CVE sets, and apply different severity thresholds. By connecting Harbor to more than one vulnerability scanner, you broaden the scope of your protection against vulnerabilities.
 
-**Vulnerability metadata** 
+**IMPORTANT**: Clair is an optional component. To be able to use Clair or to configure additional scanners, you must have enabled Clair when you installed your Harbor instance. You configure additional scanners in the Harbor interface, after you have installed Harbor. 
```

Comment:
`To be able to use Clair or to configure additional scanners, you must have enabled Clair when you installed your Harbor instance.`

This is not correct. The default `Clair` is not the precondition to enable other 3rd scanners.

Reply:
Add we should also express one point here is, if the user has their own `Clair`, they can also continue to use that `Clair` as the scanner of Harbor. The enablement is totally the same as the other scanners, configured by the admin in the scanner list.

Reply:
@steven-zou, I removed the bit about requiring Clair in order to install other scanners and added that you can use your own Clair instance in https://github.com/goharbor/harbor/pull/10029/commits/a1eac99a49aa192cb5048c5eaa8a4e2a756cd134.